### DISCUSSION TITLE
general admin rights needed to use the application?

### DISCUSSION THREAD

**Discussion by slinregtoer** [Zettlr/Zettlr#2854] [Q&A]
Hi, I want to use zettlr on my company notebook. The sysadmin team is asking if zettlr needs admin rights to run. Installation would be done with admin rights but then I have to use it with my normal account without admin rights. Is that possibel? Thx

**Comment by nathanlesage** [comment]
Zettlr should run perfectly fine without admin rights, since it doesn't do funky stuff, so go ahead! If something comes up, feel free to report a bug!

**Reply by slinregtoer** [reply]
Merci
